src/sql/magic_cmd.py

```diff
@@ -65,6 +64,8 @@ def _validate_execute_inputs(self, line):
             command, others = split[0].strip(), split[1:]
 
             if command in AVAILABLE_SQLCMD_COMMANDS:
+                if command not in ["profile"]:
+                    util.support_only_sql_alchemy_connection("%sqlcmd")
```

Comment:
looks like this will raise an error whenever we execute something that's not `profile`? why?

Reply:
`util.support_only_sql_alchemy_connection` raises error when the current connection is not SQLAlchemy ([code](https://github.com/ploomber/jupysql/blob/master/src/sql/util.py#L255)). This is included originally since dbapi connection does not support `%sqlcmd` arguments.

Since this issue is about adding support for profile, I moved `util.support_only_sql_alchemy_connection` inside the if statement so that it doesn't raise error when a user runs `%sqlcmd profile` but still raises the error when other commands are used.